tests/test_client.py

```diff
@@ -213,6 +213,8 @@ def test_check_input_parameters_beta(model_client, va_config):
             handle_unreporting,
         )
 
+    model_parameters["beta"] = 3
```

Comment:
At the start of test_client.py I set a set of initial model_parameters, but when testing bad model parameters I need to modify it. After the test is over, I change the values back so that they don't cause errors for later tests as the model_parameters are used throughout.

The other alternative is to do what is done in the other client tests where we input a different value directly as below, and go from there:
```
    with pytest.raises(ValueError):
        model_client._check_input_parameters(
            config_handler,
            office,
            estimands,
            geographic_unit_type,
            features,
            aggregates,
            fixed_effects,
            pi_method,
            {
                "beta": "bad_beta",
                "winsorize": False,
                "robust": True,
                "lambda_": 0,
            },
            handle_unreporting,
        )
```

Either option works!

Reply:
I guess I would slightly prefer the latter (for consistency)